src/item_contents.cpp

```diff
@@ -2079,23 +2257,15 @@ void item_contents::info( std::vector<iteminfo> &info, const iteminfo_query *par
     }
 }
 
-void item_contents::favorite_settings_menu( const std::string &item_name )
+void item_contents::favorite_settings_menu( item *i )
 {
-    pocket_favorite_callback cb( &contents );
-    int num_container_pockets = 0;
-    std::map<int, std::string> pocket_name;
-    for( const item_pocket &pocket : contents ) {
-        if( pocket.is_type( item_pocket::pocket_type::CONTAINER ) ) {
-            pocket_name[num_container_pockets] =
-                string_format( "%s/%s",
-                               vol_to_info( "", "", pocket.contains_volume() ).sValue,
-                               vol_to_info( "", "", pocket.max_contains_volume() ).sValue );
-            num_container_pockets++;
-        }
-    }
+
+    std::vector<item *> to_organize;
     uilist pocket_selector;
-    pocket_selector.title = item_name;
-    pocket_selector.text = keys_text() + "\n ";
```

Comment:
The "press a key to add to whitelist/blacklist" text is refering to the keys listed in `keys_text`. Removing the latter and leaving the former makes it very confusing as to what keys the player is supposed to press. The overly simple action descriptions in the keybindings menu do not help either.

Anyway, the previous UI logic is also sort of confusing, because the player may press `w` or `b` and expect some menu to show up, but instead the prompt is changed, which is always there so the player might not notice it.

Therefore I suggest we change the prompt text to something like this, so it's clearer what the hotkeys are for and that the `w` and `b` keys are supposed to switch mode instead of opening another menu.

```
Modify pocket blacklist (w switch to whitelist): i item, .....
```

Reply:
#58653